client/src/pages/Search/GridView/RestaurantCard/ImageCard.js

```diff
@@ -7,7 +7,15 @@ const classes = {
 };
 
 function ImageCard({ src }) {
-  return <img src={src} style={classes.imageCard} alt={src} />;
+  return <img 
+  src={src}
+  style={classes.imageCard}
+  onError={({ currentTarget }) => {
+    currentTarget.onerror = null; // prevents looping
+    currentTarget.src="https://images.unsplash.com/photo-1546069901-ba9599a7e63c?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8&w=1000&q=80";
+  }}
+  alt={src}
```

Comment:
alt should describe the image for screen readers, like "restaurant" or "dolphin eating cheese"